modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/pagination/Page.java

```diff
@@ -155,11 +159,12 @@ private Page(
 
 	private Page(
 		Map<String, Map<String, String>> actions, Collection<T> items,
-		Pagination pagination, long totalCount, List<Facet> facets) {
+		Pagination pagination, long totalCount,
+		Map<String, List<Facet>> facetMap) {
 
 		_actions = actions;
 		_items = items;
-		_facets = facets;
+		_facets = facetMap;
```

Comment:
Not sure about the naming. Assigning facetMap to facets may cause some trouble in a very attention to detail person 😉. May be stick with one in both cases. I prefer facets because I don't like Hungarian Notation adding type but I know that is kind of standard here

Reply:
Yeah, automatic refactor kicked in. This one can be called facets, a map can be called ending in plural if it's not already being used.